scripts/init/indicator.py

```diff
@@ -60,7 +57,8 @@ def verify_indicator_parameters(self, indicator_type_id: int, parameters: List[d
 
         return indicator_parameters
 
-    def get_data_frame(self, data_source: pandas.DataFrame, request: str, dimensions: str, measures: str):
+    @staticmethod
```

Comment:
Any reason why this should be a static method? or is it just the pylint reco?

Reply:
Pylint recommended it, but I disabled the rule and reverted this change to keep it consistent